site/app/templates/admin/admin_gradeable/AdminGradeableBase.twig

```diff
@@ -53,10 +55,10 @@
 <script>
 
     let adminGradeableNavTab = 0;
-    let navItemsStatus = new Map([['0',true],['1',true],['2',true],['3',true],['4',true]]);
+    var navItemsStatus = new Map([['0',true],['1',true],['2',true],['3',true],['4',true],['5',false]]);
```

Comment:
Is there a reason this being changed from `let` to `var`. We typically want to use `let` since JS will scope `var` to the most immediate function body, `let` will be scoped to the most immediate set of brackets.

JS scoping can sometimes get weird causing unexpected errors so using `const` or `let` instead is usually better - here it might not matter as much since they're globally defined in that script

Reply:
I thought github deleted my comment here but i guess it posted it twice - my bad